spec/cassette/cache_spec.rb

```diff
@@ -2,18 +2,54 @@
 
 describe Cassette::Cache do
   subject do
-    c = Class.new
-    c.send(:include, Cassette::Cache)
-    c.new
+    cached()
+  end
+
+  after do
+    described_class.instance_variable_set(:@backend, nil)
   end
 
-  describe 'backend' do
+  describe '.backend' do
+    it 'defaults to rails backend' do
+      rails = double('Rails')
+      allow(rails).to receive(:cache).and_return(rails)
```

Comment:
It is working, but I think it is a little weird because the method `cache` is returning the same value as the Rails const.

What about like this?
```ruby
rails_cache = double('rails_cache')
allow(Rails).to receive(:cache).and_return(rails_cache)

expect(described_class.backend).to eql(rails_cache)
```

Reply:
We don't have the `Rails` constant in this project since rails is not a hard dependency, hence the const emulation